src/SprykerEco/Service/ComputopApi/ComputopApiService.php

```diff
@@ -64,12 +64,12 @@ public function extractResponseHeader(array $plaintextResponseHeader, $method):
         $header = $this->getFactory()->createComputopApiConverter()->extractResponseHeader($plaintextResponseHeader, $method);
 
         $expectedMac = $this->getHashValue(
-            $this->getFactory()->createComputopApiMapper()->getMacResponseEncryptedValue($header)
+            $this->getFactory()->createComputopApiMapper()->getMacResponseEncryptedValue($header),
```

Comment:
in general, service should only contain a single call
please introduce the respective class and move this code there
same for case below

Reply:
Fixed